### PR TITLE
Created initial options page

### PR DESCRIPTION
**Pull request by brianlee389**
Here's just an initial of what exists for the settings. Just wanted to put this here if you wanted to see what it would like.

### CODE REVIEW

File: inc/snowball-config.php [hunk 1]
```diff
@@ -170,5 +170,60 @@ function snowball_enable_front_page($query) {
...
+
+  /*
+   *  Use the add_options_page function
+   *  add_options_page( $page_title, $menu_title, $capability, $menu-slug, $function ) 
+   *
+  */
+
+  add_options_page(
+    'Snowball Settings',
+    'Snowball Settings',
```
Comment (thomaspark) [inline inc/snowball-config.php]: Let's call the menu title just 'Snowball' to keep it consistent with the rest.

File: inc/snowball-config.php [hunk 2]
```diff
@@ -170,5 +170,60 @@ function snowball_enable_front_page($query) {
...
+    'manage_options',
+    'snowball-slug',
+    'snowball_options_page'
+  );
+
+}
+add_action( 'admin_menu', 'snowball_menu' );
```
Comment (thomaspark) [inline inc/snowball-config.php]: Fix whitespace to in parens to keep consistent codebase. Same for the rest of the parens.

File: inc/snowball-config.php [hunk 3]
```diff
@@ -170,5 +170,60 @@ function snowball_enable_front_page($query) {
...
+
+function snowball_options_page() {
+
+  if( !current_user_can( 'manage_options' ) ) {
+
+    wp_die( 'You do not have suggicient permissions to access this page.' );
```
Comment (thomaspark) [inline inc/snowball-config.php]: "suggicient" -> "sufficient"

File: inc/snowball-options.php [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<div class="wrap">
+	<h2>Snowball Options</h2>
```
Comment (thomaspark) [inline inc/snowball-options.php:2]: Call this "Snowball Settings" to keep it consistent.

Fix tabs to 2 spaces.

Add newline at end of file.

**Comment by thomaspark** [comment]
Looks good, just some small tweaks and I'll merge it.

**Comment by brianlee389** [comment]
I made the all the revisions and pushed again.

**Pull request merged**